backend/app/data/survey.json

```diff
@@ -64,14 +64,6 @@
                         {
                             "value": "U",
                             "text": "Undergraduate Student"
-                        },
-                        {
-                            "value": "MScAC",
-                            "text": "MScAC"
-                        },
-                        {
-                            "value": "PD",
-                            "text": "Postdoctorate Fellow"
```

Comment:
Why are these removed?

Reply:
I'm a clown and somehow deleted this. Fixed in new commit.